### DISCUSSION TITLE
When will the next build be released?

### DISCUSSION THREAD

**Discussion by Anders-Johansen** [rism-digital/verovio#2872] [Q&A] [answered]
Hi all,
I am using Verovio in my website project to render a lot of examples and tasks to our students. I am trying to make a new release of the site, but need the new features on auto sizing. Do any of you know, when the js-version will be built?
Thank you for amazing work on Verovio!
Best regards,
Anders Johansen

**Comment by lpugin** [comment, marked as answer]
We typically try to have a release about every two months. This time it is taking a bit longer but I will try to prepare one within the next week.

**Reply by Anders-Johansen** [reply]
Thank you, that would be really awesome!
